### ISSUE TITLE
Selection location filter first, then Country View results in multiple location search?

### ISSUE THREAD

**Issue by programmieraffe** [hbz/oerworldmap#1784]
http://recordit.co/fCidf9Famz
(Screenshot taken in beta)

Did not find the issue, thought I created it before -> Location filter should redirect to country view?

**Comment by literarymachine** [comment]
> Location filter should redirect to country view?

Nope, the location filters allows to select multiple countries! But we indeed have to unset the `location` filter when navigating to a country (this was not an issue until #1755).

**Comment by programmieraffe** [comment]
Ahh, never noticed that this is a "multiple values filter" 👍 

Problem I see: Only in Country View the user can choose the region filter than - we should discuss how we communicate that to the user? Or if we recommend to the user to switch to country view, if only one location is selected?

(Multi-Country-View is for experts?!)

**Comment by literarymachine** [comment]
For now, we will simply hide the `location` / `region` filter and implement geographic filtering solely via the map. See #1799.

**Comment by literarymachine** [comment]
> For now, we will simply hide the location / region filter and implement geographic filtering solely via the map

Deployed to beta.

**Comment by programmieraffe** [comment]
+1
